user-guide/Standard_Apps/Monitoring_Solutions/Nevion Video IPath/Using_NevionVideoIPath_App.md

```diff
@@ -0,0 +1,62 @@
+---
+uid: How_to_NevionVideoIPath_App
+---
+
+# Using the Nevion Video IPath app
+
+To access the application:
+
+1. Go to `http(s)://[DMA name]/root`.
+
+1. Select *Nevion Video IPath* to start using the application.
+
+   ![Nevion Video IPath](~/user-guide/images/NevionAppLogo.png)
+
+## The Nevion Video IPath interface
+
+The sidebar on the left of the Nevion Video IPath application contains two buttons:
+
+| Button | Page description |
+|:--:|--|
+| ![Connect](~/user-guide/images/Nevion_Connect.png) | Opens the [*Connect* page](#connect-page), which displays profiles, sources, and destinations. The connection between a source and a destination establishes a service. |
+| ![Services](~/user-guide/images/Nevion_Services.png) | Opens the [*Services* page](#services-page), which provides an overview of all available services. |
+
+## Connect page
+
+The *Connect* page allows you to establish a connection between sources and destinations.
+
+![Overview page](~/user-guide/images/Nevion_OverviewPage.png)
+
+To establish a connection:
+
+1. Select a specific **profile** to see compatible sources and destinations.
+
+   These profiles correspond to logical, predefined sets of configurations that apply to specific types of connections.
+
+1. Select a **source and destination** to connect.
+
+   Nevion allows you to use tags to categorize, filter, and organize sources (inputs) and destinations (outputs).
+
+1. Click **Connect** to link the source and destination.
+
+   The *Last Connection Status* will show whether the connection was successful.
+
+   Once a source and destination have been connected, you can disconnect them again with the **Disconnect** button.
+
+   Alternatively, you can also use the **Schedule** button.
+
+## Services page
+
+The *Services* page provides an overview of the services in the system, which correspond to established connections between sources and destinations.
+
+![Overview page](~/user-guide/images/Nevion_ServicesPage.png)
+
+In addition to this overview, services can easily be deleted from this table.
```

Comment:
Can you describe in more detail how to do this? Also note that it's best to avoid terms like "easily" in technical documentation, as these can trigger frustration in users who are struggling to figure out how something works.

Reply:
Understood. Well, in that table there is a button in the row to delete the service. My new proposal: "In addition to this overview, services can be conveniently deleted from this table using the cross-mark button."